### PR TITLE
Standards export

### PR DESCRIPTION
**Pull request by mrjoshida**
Addressing common request for a downloadable standards spreadsheet.

### CODE REVIEW

File: standards/views.py [hunk 1]
```diff
@@ -37,6 +38,26 @@ def by_curriculum(request, slug):
...
+    writer.writerow(['curriculum', 'unit', 'lesson #', 'lesson name',
+                     'standard framework', 'standard', 'cross curricular opportunity'])
+    for unit in curriculum.units:
+        for lesson in unit.lessons:
+            for standard in lesson.standards.all():
+                writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                                 standard.framework.slug, standard.shortcode, ''])
+            for standard in lesson.opportunity_standards.all():
+                writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                                 standard.framework.slug, standard.shortcode, 'X'])
```
Comment (Hamms) [inline standards/views.py]: nit: is there a specific reason to use `''`/`'X'` rather than `true`/`false`?

File: standards/views.py [hunk 2]
```diff
@@ -53,6 +74,26 @@ def by_unit(request, slug, unit_slug):
...
+    writer.writerow(['curriculum', 'unit', 'lesson #', 'lesson name',
+                     'standard framework', 'standard', 'cross curricular opportunity'])
+    for lesson in unit.lessons:
+        for standard in lesson.standards.all():
+            writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                             standard.framework.slug, standard.shortcode, ''])
+        for standard in lesson.opportunity_standards.all():
+            writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                             standard.framework.slug, standard.shortcode, 'X'])
+    return response
```
Comment (Hamms) [inline standards/views.py]: it might make things simpler to give Lesson something like a `to_csv` method that can be used here and above

Comment (mrjoshida) [reply standards/views.py]: This is a min bar to address the immediate need. I plan to come back here and flesh out a better solution later.

File: standards/views.py (hunk 1 above)
Comment (mrjoshida) [reply standards/views.py]: switched

**Pull request merged**